dashboard/test/controllers/registrations_controller/set_age_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+module RegistrationsControllerTests
+  #
+  # Tests over PATCH /users/set_age
+  #
+  class SetAgeTest < ActionDispatch::IntegrationTest
+    test "set_age does nothing if user is not signed in" do
+      User.any_instance.expects(:update).never
+      patch '/users/set_age', params: {user: {age: '20'}}
+      assert_response :forbidden
+    end
+
+    test "set_age does nothing if user age is already set" do
+      User.any_instance.expects(:update).never
+      student = create :student
+      refute student.age.blank?
+
+      sign_in student
+      patch '/users/set_age', params: {user: {age: '20'}}
+      assert_response :success
```

Comment:
This doesn't actually assert that is "does nothing". If that's something we want to verify, we should set the initial age to a known value, try to set it to a different value through the API, and then assert it's still the original.

Reply:
That's a good point - I was leaning on `User.any_instance.expects(:update).never` but there are other ways to change an attribute.  I'll update this test.

Reply:
Ah, also I didn't notice that expectation when I looked earlier.